versionutils/version.go

```diff
@@ -337,3 +342,21 @@ func MatchesRegex(tag string) bool {
 func GetImageVersion(version *Version) string {
 	return version.String()[1:]
 }
+
+func Index(versions []Version, v Version) int {
```

Comment:
I think this is unused

Reply:
If we can leave this in, it may be useful in the future?